src/main/scala/components/csr.scala

```diff
@@ -0,0 +1,506 @@
+/* Describes register file that maintains machine state */
+
+package dinocpu
+
+import chisel3._
+import collection.mutable.LinkedHashMap
+import chisel3.util._
+import chisel3.util.BitPat
+import chisel3.util.experimental.BoringUtils
```

Comment:
Needed? It would be nice to not use this. So far, it's not been necessary.

Reply:
removed them except 1st 2 which are needed